src/Microsoft.OData.Core/JsonLight/ODataJsonLightContextUriParser.cs

```diff
@@ -299,7 +299,7 @@ private ODataPayloadKind ParseContextUriFragment(string fragment, Func<IEdmType,
                 {
                     detectedPayloadKind = ODataPayloadKind.ServiceDocument;
                 }
-                else if (fragment.Equals(ODataConstants.EntityReferenceCollectionSegmentName + "(" + ODataConstants.EntityReferenceSegmentName + ")"))
+                else if (fragment.Equals(ODataConstants.CollectionPrefix + "(" + ODataConstants.EntityReferenceSegmentName + ")"))

```

Comment:
The collection prefix "Collection" isn't specific to usage by an EntityReference, or even within a segment; it's a prefix that can be used in multiple places (including type names). The EntityReference ($ref) is more limited in usage, so its current name is more appropriate.

---
In reply to: [127301964](https://github.com/OData/odata.net/pull/894#discussion_r127301964) [](ancestors = 127301964)